Cargo.toml

```diff
@@ -69,10 +72,11 @@ simple_logger = { version = "2.1.0", default_features = false }
 softbuffer = "0.3.0"
 
 [target.'cfg(target_os = "android")'.dependencies]
-# Coordinate the next winit release android-activity 0.5 release
-android-activity = "0.5.0-beta.1"
-ndk = "0.8.0-beta.0"
-ndk-sys = "0.5.0-beta.0"
+# https://github.com/rust-mobile/android-activity/pull/128
+# android-activity = "0.5.0"
+android-activity = { git = "https://github.com/MarijnS95/android-activity", rev = "720e5a3" }
```

Comment:
Since we're using a git dependency, should this PR be marked as draft for now?

Reply:
Yeah, git dependencies are not allowed for stable releases.

Reply:
It is already marked as `Depends on`, and I'd rather push forward on getting a release out some time soon.

@rib if you can, the NDK crate is now ready and `android-activity` can go out of beta too.

Reply:
@MarijnS95 distributions don't package crates having `git` dependencies, even for non-native platforms. I can't do a release with the `git` dependencies.

Maybe sending email to @rib will work if they are missing.

Reply:
@kchibisov we are thinking and saying the same.

I'm just not the release manager for `android-activity`, and @rib doesn't seem to have the `cargo publish` step automated in the CI with a token in GitHub secrets so I won't be able publish a release in his name either way.

Regardless, it hasn't been 24h yet since I published the `ndk` and it's still weekend so I wouldn't call it missing, just unfortunate timing.

Reply:
Anyway, I've converted this PR to "draft" as that seems to be the main point of contention (...?).

Reply:
>@kchibisov we are thinking and saying the same.

Ah, release out of **android activity**, not winit. I've opened an issue on their tracker. At least it seems like issues are being read quite often.

Reply:
I'm sure he's fully aware of the release schedule, just unfortunate timing on my end and it's quite a long trail (release `raw-window-handle` -> `ndk(-sys)`, `android-activity`, `winit`, all owned and maintained by different people).